homestar-invocation/src/task/config.rs

```diff
@@ -12,10 +13,16 @@ const MEMORY_KEY: &str = "memory";
 const TIMEOUT_KEY: &str = "time";
 
 /// Resource configuration for defining fuel quota, timeout, etc.
-#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize, JsonSchema)]
+#[schemars(
+    rename = "resources",
+    description = "Resource configuration for fuel quota, memory allowance, and timeout"
```

Comment:
minor: so no `.`'s on this stuff?

Reply:
The periods look a bit strange when rendered as a JSON value, so left them off. Not sure if I was consistent with that though, can clean up is so.

Reply:
yeah, double-check on consistency.

Reply:
We're going to hold off on being consistent here for the moment to avoid extra schema description noise. Will re-visit in the future.